README.md

```diff
@@ -48,20 +48,20 @@ The affected-paths library can be found on [MavenCentral][1]:
 ### Gradle
 Groovy
 ```groovy
-implementation 'com.squareup.affected.paths:affected-paths-core:0.1.1'
+implementation 'com.squareup.affected.paths:affected-paths-core:0.1.2'
```

Comment:
ergonomic suggestion for you: write out the version everywhere as `<<latest>>` and have just one reference somewhere to that, maybe even just the changelog.